modules/omx/omx.h

```diff
@@ -0,0 +1,30 @@
+#include <IL/OMX_Core.h>
+#include <IL/OMX_Video.h>
+#include <IL/OMX_Broadcom.h>
+#include <pthread.h>
+
+
+struct vidframe* frame;
+
+
+struct omx_state {
+	pthread_mutex_t omx_mutex;
+
+	OMX_HANDLETYPE video_render;
+	OMX_BUFFERHEADERTYPE** buffers;
+	int num_buffers;
+	int current_buffer;
```

Comment:
Need to do more research on that. OpenMAX returns a certain amount of buffers for a given port definition, probably to do some kind of double/triple-buffering.
But it worked fine with just using one buffer, so i didn't bother to implement that yet. Might be necessary to support the bigger resolutions, where one frame spans across multiple OMX buffers.